incubator-tools/reorder_document_page_based_on_unique_strings/reorder_document_page_based_on_unique_strings.ipynb

```diff
@@ -0,0 +1,363 @@
+{
```

Comment:
<p>I'm not sure I understand what this notebook does or what the use case is</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/700/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
This tool uses OCR to reorder PDF pages based on unique strings (logical identifiers) provided by the user, ideal for organizing out of sequence PDF pages into their correct order.